### ISSUE TITLE
Cancel deployment

### ISSUE THREAD

**Issue by nbarrera** [aws/copilot-cli#2544]
Hi guys,

I 've just experimented starting a deployment which introduced changes that made the container fail the health check.

So I just hit `ctr+c` and interrupted the `copilot` CLI, fixed the bug then tried to deploy again but the newly issued command fails prompty with:

```
✘ Proposing infrastructure changes for stack xxx-test-site 
✘ deploy service: stack xxx-test-site is currently being updated and cannot be deployed to
```

So when I show status

```
% copilot svc status
Found only one deployed service site in environment test
Task Summary

  Running      ██████████  1/1 desired tasks are running
  Deployments  ░░░░░░░░░░  0/1 running tasks for primary (rev 5)
```

This means that the deployment with the bug was still trying to be done.

I had to wait a long time until all resources were freed up and was able to make an `svc deploy` again.

Is there any way to cancel a deployment that's currently running?

This could be useful to avoid a wrong deployment from taking place, or to avoid this long retries of an impossible to succeed deployment.

**Comment by Lou1415926** [comment]
Hi @nbarrera !

Yon can cancel the stack update through CloudFormation console by clicking "Stack actions" and then "cancel stack updates"
<img width="794" alt="Screen Shot 2021-07-01 at 11 38 10 AM" src="https://user-images.githubusercontent.com/79273084/124159513-dc241580-da60-11eb-9216-3ba6db1676d1.png">

After the rollback is completed, you can run `svc deploy` again.

Hope this helps!!

**Comment by nbarrera** [comment]
Sorry I didn't know about that
thanks!
👍

**Comment by runekaagaard** [comment]
Thank you, took me WAY too long to find this. As a copilot beginner a single wrong setting costs about an hour of waiting time without it. A user friendly way that other apps use are something like this:

1. After copilot deploy display: `To rollback the current deploy press ctrl+c once.`
2. First `ctrl+c`: Show prompt: `Do you want to rollback the current deploy? (y/n).`
3. Second `ctrl+c`: Show prompt: `Are you sure you want to force quit the current deployment, bla bla? (y/n)`.
4. Third `ctrl+c`: Kill everything

**Comment by efekarakus** [comment]
Thanks @runekaagaard that UX looks great 🤩

**Comment by sxalexander** [comment]
An aws command line approach to cancel a running stack update is:

```
$ aws cloudformation cancel-update-stack --stack-name $STACK_NAME
```

Where $STACK_NAME is what you see in the output after `$ copilot svc deploy` which typically looks like

```
$ copilot svc deploy
Updating the infrastructure for stack [STACK_NAME IS HERE]
```

**Comment by KollaAdithya** [comment]
Hey @sxalexander !

Currently we are actively working on this feature. We will post here once this is released. stay tuned!

**Comment by KollaAdithya** [comment]
Hello @nbarrera and @sxalexander !

Ctrl-C feature is released in [v1.30](https://github.com/aws/copilot-cli/releases/tag/v1.30.0) 🎉 . You can now cancel your stack update to rollback to previous configuration.
